modules/nf-core/amulety/esm2/main.nf

```diff
@@ -0,0 +1,51 @@
+process AMULETY_ESM2 {
+    tag "$meta.id"
+    label 'process_medium'
+    label 'process_gpu'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/amulety_igblast:b2a7736f645c40e5':
+        'community.wave.seqera.io/library/amulety_igblast:659eaa872785adeb' }"
+
+    input:
+    tuple val(meta), path(tsv)
+    val(chain)
+
+    output:
+    tuple val(meta), path("*_esm2.tsv"), emit: embedding
```

Comment:
```suggestion
    tuple val(meta), path("*.tsv"), emit: embedding
```
The guidelines state that custom suffixes should be avoided to allow full flexibility to the pipeline developers.